src/jit/lclvars.cpp

```diff
@@ -495,8 +496,6 @@ void                Compiler::lvaInitRetBuffArg(InitVarDscInfo *    varDscInfo)
             }
         }
 
-        assert(genMapIntRegNumToRegArgNum(varDsc->lvArgReg) < MAX_REG_ARG);
```

Comment:
I can change this assert to 
        assert(isValidIntArgReg(varDsc->lvArgReg));